src/Backend/Modules/ContentBlocks/Actions/Add.php

```diff
@@ -10,91 +10,72 @@
  */
 
 use Backend\Core\Engine\Base\ActionAdd as BackendBaseActionAdd;
-use Backend\Core\Engine\Authentication as BackendAuthentication;
 use Backend\Core\Engine\Model as BackendModel;
-use Backend\Core\Engine\Form as BackendForm;
-use Backend\Core\Engine\Language as BL;
-use Backend\Modules\ContentBlocks\Engine\Model as BackendContentBlocksModel;
+use Backend\Modules\ContentBlocks\ContentBlock\ContentBlock;
+use Backend\Modules\ContentBlocks\ContentBlock\ContentBlockType;
+use Backend\Modules\ContentBlocks\ContentBlock\CreateContentBlock;
 
 /**
  * This is the add-action, it will display a form to create a new item
  */
 class Add extends BackendBaseActionAdd
 {
     /**
-     * The available templates
-     *
-     * @var	array
-     */
-    private $templates = array();
-
-    /**
      * Execute the action
      */
     public function execute()
     {
         parent::execute();
-        $this->templates = BackendContentBlocksModel::getTemplates();
-        $this->loadForm();
-        $this->validateForm();
-        $this->parse();
-        $this->display();
-    }
 
-    /**
-     * Load the form
-     */
-    private function loadForm()
-    {
-        $this->frm = new BackendForm('add');
-        $this->frm->addText('title', null, null, 'form-control title', 'form-control danger title');
-        $this->frm->addEditor('text');
-        $this->frm->addCheckbox('hidden', true);
+        $createContentBlockCommand = new CreateContentBlock();
```

Comment:
no need to suffix the var name with `command`. Your class name is written as an imperative, which already shows you it is a command.